frontend/public/components/app-contents.tsx

```diff
@@ -115,7 +115,7 @@ const AppContents_: React.FC<AppContentsProps> = ({ activePerspective, flags })
 
           <Route path={['/all-namespaces', '/ns/:ns']} component={RedirectComponent} />
           <LazyRoute
-            path="/dashboards"
+            path="/overview"
```

Comment:
I'm undecided on this because it will break old links to old route `/overview/:namespace`, which used to be the project overview. We might leave the URL as-is and just update the labels in the UI.

Reply:
FYI @rawagner after some more discussion I think we've agreed to not break the URL right now to avoid bookmarks going to a weird place.

Reply:
ok, the change to url was removed